example/prometheus/main.go

```diff
@@ -0,0 +1,75 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"log"
+	"net/http"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+
+	"go.opentelemetry.io/otel/attribute"
+	otelprom "go.opentelemetry.io/otel/exporters/prometheus"
```

Comment:
Its somewhat unfortunate that everyone will have to rename the import to use the exporter.  I seem to recall it makes it harder to use editors that automatically add imports.  Should we intentionally choose a different package name from `prometheus`?  e.g. `go.opentelemetry.io/otel/exporters/prometheus/otelprom`?

Doesn't block this PR

Reply:
`go.opentelemetry.io/otel/exporters/otelprom` seems reasonable to me.

Reply:
I have a slight preference for spelling it out `otelprometheus`, to be clear.

Reply:
It looks like `github.com/prometheus/client_golang/prometheus` is only imported to set up the registry to use with the server provided by `promhttp`.  I don't think this was required with the old exporter implementation as it would return a handler directly.  If the exporter provides a mechanism to obtain an `http.Handler`, filling the same purpose as `serveMetrics()` here, this might be a non-issue.

Reply:
I'm going to open an issue to track this package name change. Given what we have in `new_sdk/main` replaces what is on `main`, if we decide to change in the next milestone, or add the mentioned method, we will have an equivalent state. Therefore, I'll include the issue the beta milestone.

Reply:
Tracking with https://github.com/open-telemetry/opentelemetry-go/issues/3172